public/graph.js

```diff
@@ -917,6 +918,16 @@ class GraphController {
             updateViewOneDay(yLimitReceived, yLimitSent, yLimitFailed);
         });
 
+        d3.select("#timeFrame").on("change", function() {
```

Comment:
The misalignment, especially on day->48 hours, is so visible I wouldn't want us to merge this then resolve in a diff branch so probably better to solve it here? 
So probably the first place to start this is changing from using the data to determine the bar width? especially on the 1-day view? Having a standard width and the use the data to set the bar heights.

Reply:
I have updated the graph bar width and x attributes and enabled little space between bars to avoid misalignment of bars as in issue  #115. 
Note: I have made the x axis ticks to be at the center of the bar graphs,I have noticed enabling them to be at the start of a bar chart makes it shift when the timeframe is switched altering the correctness of the graph

Reply:
let switch back to left-aligned ticks, and then fix the width mis-alignments.

Reply:
I have noticed there is a problem with my implementation and there is no reason to calculate the x position and bar width. I will update them to default.